integration-test/tests/cms/eventsfilters.test.ts

```diff
@@ -17,3 +17,13 @@ test('GET /api/cms/v1/eventsfilters returns success response code', async ({ req
     }
   }
 });
+test('should return 401 Unauthorized when no authentication header is provided', async ({ request }) => {
+  const response = await request.get('/api/cms/v1/events/filters', {
+    headers: {}, // Override global headers to simulate no authentication
+  });
+});
+test('should return 401 Unauthorized when invalid key is provided', async ({ request }) => {
+  const response = await request.get('/api/cms/v1/events/filters', {
+    headers: { 'X-API-KEY': 'invalid_key' }, // Override global headers to simulate no authentication
```

Comment:
I think having the test name `should return 401 Unauthorized when invalid key is provided` is clear and you don't need to add this inline comment